upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -542,6 +542,32 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 		}
 	}
 
+	components := b.Cluster.Spec.Components
+
+	if components != nil {
+		nth := components.NodeTerminationHandler
+
+		if nth != nil && fi.BoolValue(nth.Enabled) {
+
+			key := "node-termination-handler.aws"
+			version := "1.7.0"
+
+			{
+				location := key + "/k8s-1.12.yaml"
+				id := "k8s-1.12"
+
+				addons.Spec.Addons = append(addons.Spec.Addons, &channelsapi.AddonSpec{
+					Name:              fi.String(key),
+					Version:           fi.String(version),
+					Selector:          map[string]string{"k8s-addon": key},
+					Manifest:          fi.String(location),
+					KubernetesVersion: ">=1.12.0",
```

Comment:
If this is going into kops 1.19, you'll need to do something for k8s 1.11. Prohibiting it through API validation would be fine.

Reply:
My mistake. I thought perhaps it was 1.12. I didn't actually try this on any older versions if there are manifest incompatibilities. I can quickly test that part.